CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 ### 0.4.2 (Next)
 
+* [#45](https://github.com/dblock/strava-ruby-client/pull/45): Added `Gear#frame_type` `Gear#retired` `Gear#weight`- [@deuber](https://github.com/deuber).
```

Comment:
Added `Gear#frame_type`, `#retired`, and `#weight`.

Reply:
added missing space, yeah was very hidden, thanks